app/javascript/.storybook/preview.js

```diff
@@ -50,3 +50,12 @@ const themeSwitcherDecorator = (storyFn) => (
 );
 
 addDecorator(themeSwitcherDecorator);
+
+addParameters({
```

Comment:
This is what does the sorting. See the documentation for [Sorting Stories](https://storybook.js.org/docs/configurations/options-parameter/#sorting-stories) for more info.